app/Controllers/Account/SecurityController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Controllers\Account;
+
+use App\Controllers\BaseController;
+use CodeIgniter\HTTP\RedirectResponse;
+use CodeIgniter\HTTP\Exceptions\HTTPException;
+use CodeIgniter\Shield\Authentication\AuthenticationException;
+use CodeIgniter\Shield\Models\RememberModel;
+use ReflectionException;
+
+class SecurityController extends BaseController
+{
+    /**
+     * Manage the user's account security.
+     */
+    public function index()
+    {
+        return $this->render('account/security/security', [
+            'user' => auth()->user(),
+            'logins' => auth()->user()->logins(7),
+            'agent' => $this->request->getUserAgent(),
+            'isRemembered' => model(RememberModel::class)->where('user_id', user_id())->countAllResults() > 0,
+        ]);
+    }
+
+    /**
+     * Logs the user out of all sessions, by deleting all remember-me tokens.
+     * Logs the user out and redirects the user to the login page.
+     */
+    public function logoutAll()
+    {
+        try {
+            auth()->forget(auth()->user());
```

Comment:
I don't know exactly what you mean, but if you want to remove remember-me user, I think the following code snippet is good.
```php
use CodeIgniter\Shield\Models\RememberModel;
// ...

$rememberModel = model(RememberModel::class);

/** @var User::class $user */
$user = auth()->user();

$rememberModel->purgeRememberTokens($user);
```

Reply:
That's basically what the `forget()` method on the Session Authenticator does. 

https://github.com/codeigniter4/shield/blob/50fb634742c2b4379c51855ae7175d77c34eeac3/src/Authentication/Authenticators/Session.php#L854

Reply:
Thanks, I hadn't seen this one.